query-engine/core/src/trace_helpers/mod.rs

```diff
@@ -47,10 +48,24 @@ fn span_to_json(span: &SpanData) -> Value {
         "name": name,
         "start_time": span.start_time.duration_since(SystemTime::UNIX_EPOCH).unwrap().as_millis().to_string(),
         "end_time": span.end_time.duration_since(SystemTime::UNIX_EPOCH).unwrap().as_millis().to_string(),
-        "attributes": attributes
+        "attributes": attributes,
+        "links": create_link_json(span)
```

Comment:
Not seeing this on the client side

Reply:
Turns out it was an installation problem, sha of migration engine made install fail silently.